NuRadioReco/modules/channelAddCableDelay.py

```diff
@@ -0,0 +1,23 @@
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units
+import logging
+
+
+class channelAddCableDelay:
```

Comment:
There is a new model that calculates the efield per channel (to get the "simChannel" voltage traces). This module does not include the cable delays. I found it more logic to have a separate module for it. 
In case of the efieldToVoltageConverter this is not possible because we don't have the intermediate step of calculating simchannel traces.

Reply:
Okay, makes sense